### DISCUSSION TITLE
3 PCB Noirs disponible pour La France / 3 Black PCBs available for France

### DISCUSSION THREAD

**Discussion by Treuze63** [FundamentalFrequency/.github#45] [General]
Bonjour,
Tout d'abord, merci pour ce projet qui permet de m'occuper au niveau bricolage et musique!
J'ai donc trois PCB disponible pour la France, que j'ai publié sur LEBONCOIN pour plus de facilité (recherchez LMN 3)
(Je ne prend pas de marge dessus, vous payez juste les frais de port en plus)
Longue vie au LMN 3!

(google Trad)
Hello,
First of all, thank you for this project which keeps me busy with DIY and music!
So I have three PCBs available for France, which I published on LEBONCOIN for convenience (search for LMN 3)
(I don't take a margin on it, you just pay extra shipping)
Long live to the LMN 3!

**Comment by pixiesue** [comment]
Bonjour Treuze, 
Je suis content de voir un autre francophone ici !
J'ai déjà un pcb mais je suis intéressé pour des découpes laser ou impressions 3d pour la case si tu aas des idées pour faire ça en local.
Par avance merci, je suis sur le discord non officiel aussi.
Bon courage pour les soudures et le setup.

**Reply by Treuze63** [reply]
Merci, je suis loin d'avoir commencé! Ravis également !  Je suis en train de regrouper les différentes épaisseurs des fichiers 3D du boitier pour avoir un seul bloc, et par la suite les splitter en 2 parties pour que ça rentre sur un plateau d'impression 3D 30cm par 30cm, je pourrais partager les fichiers par la suite si besoin.

(google trad)
Thank you, I'm far from having started! Happy too! I am in the process of grouping the different thicknesses of the 3D files for the box, (to have a single block), and then splitting them into 2 parts, to fits on a 30cm by 30cm 3D printing plate, I could share the files if needed.

**Reply by pixiesue** [reply]
Someone from the unofficial discord shared that:
https://www.thingiverse.com/thing:5426597/files

it's still wip but it might help you.

**Reply by Treuze63** [reply]
Bien mieux que ce que j'ai commencé à faire, j'aurais du regarder avant sur thingiverse, merci pour le partage! (je pense venir sur le discord du coup ^^)

**Comment by Treuze63** [comment]
Pour vous informer sur mon stock, il me reste 2 cartes, merci la communauté lmn-3 !!
